Microsoft.Toolkit.Uwp.UI.Controls/SurfaceDialTextboxHelper/SurfaceDialTextboxHelper.cs

```diff
@@ -0,0 +1,403 @@
+﻿// ******************************************************************
+// Copyright (c) Microsoft. All rights reserved.
+// This code is licensed under the MIT License (MIT).
+// THE CODE IS PROVIDED “AS IS”, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+// INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.
+// IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+// DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+// TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH
+// THE CODE OR THE USE OR OTHER DEALINGS IN THE CODE.
+// ******************************************************************
+
+using System.Diagnostics;
+using Windows.Foundation.Metadata;
+using Windows.UI.Input;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.UI.Xaml.Input;
+
+namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Helper class that provides attached properties to enable any TextBox with the Surface Dial. Rotate to change the value by StepValue between MinValue and MaxValue, and tap to go to the Next focus element from a TextBox
+    /// </summary>
+    public static class SurfaceDialTextboxHelper
+    {
+        /// <summary>
+        /// The Surface Dial controller instance itself
+        /// </summary>
+        private static RadialController _controller;
+
+        /// <summary>
+        /// A default menu item that will be used for this to function. It will automatically be cleaned up when you move away from the TextBox, and created on Focus.
+        /// </summary>
+        private static RadialControllerMenuItem _stepTextMenuItem;
+
+        /// <summary>
+        /// The textbox itself needed to refernece the current TextBox that is being modified
+        /// </summary>
+        private static TextBox _textBox;
+
+        /// <summary>
+        /// Gets or sets the controller for the Surface Dial. The RadialController can be set from your app logic in case you use Surface Dial in other custom cases than on a TextBox.
+        /// This helper class will do everything for you, but if you want to control the Menu Items and/or wish to use the same Surface Dial insta
+        /// This is the property for the static controller so you can access it if needed.
+        /// </summary>
+        public static RadialController Controller
+        {
+            get
+            {
+                return _controller;
+            }
+
+            set
+            {
+                _controller = value;
+            }
+        }
+
+       /// <summary>
+        /// If you provide the Controller yourself, set this to true so you won't add new menu items.
+        /// </summary>
+        public static readonly DependencyProperty ForceMenuItemProperty =
```

Comment:
can you move all DP at the beginning of the file (my bad I forgot to mention it earlier)